packages/sideloader/build.sh

```diff
@@ -0,0 +1,43 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Dadoum/Sideloader
+TERMUX_PKG_DESCRIPTION="Open-source cross-platform iOS app sideloader"
```

Comment:
I'm not sure how useful this will be given that Termux is a terminal and packaging environment for Android.
A more generally applicable "example project" may be `tilix`[^1].

[^1]: https://github.com/gnunn1/tilix